wagtail/wagtailadmin/menu.py

```diff
@@ -104,7 +105,7 @@ def __init__(self, label, menu, **kwargs):
 
     @property
     def media(self):
-        return Media(js=['wagtailadmin/js/submenu.js']) + self.menu.media
+        return Media(js=[static('wagtailadmin/js/submenu.js')]) + self.menu.media
```

Comment:
I'm not sure that using of templatetags in Python code is good idea. Maybe something like this will be better?
```python
from django.contrib.staticfiles.storage import staticfiles_storage
staticfiles_storage.url('wagtailadmin/js/submenu.js')
```